Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM rabbitmq:3.6-management
+
+RUN apt-get update
```

Comment:
@tkuldeep In Docker, every RUN step creates an image layer. You should consider combining some of these logically using `&&`. Think of layers that will not change often at the top followed by layers that change more frequently at the bottom. This way, when rebuilding the docker image, the top few layers will be reused from cache.